robotbase/src/main/java/com/team2357/frc2022/commands/FailsafeCommand.java

```diff
@@ -0,0 +1,30 @@
+package com.team2357.frc2022.commands;
+
+import com.team2357.lib.commands.CommandLoggerBase;
+import com.team2357.lib.subsystems.ClosedLoopSubsystem;
+
+import edu.wpi.first.wpilibj2.command.CommandScheduler;
+
+public class FailsafeCommand extends CommandLoggerBase{
+    private boolean m_failsafeActive;
+    private ClosedLoopSubsystem m_subsystem;
+    
+    public FailsafeCommand(boolean failsafeActive, ClosedLoopSubsystem subsystem) {
+        m_failsafeActive = failsafeActive;
+        m_subsystem = subsystem;
+    }
+
+    @Override
+    public void initialize() {
+        super.initialize();
+
+        m_subsystem.setClosedLoopEnabled(m_failsafeActive);
+
+        CommandScheduler.getInstance().cancelAll();
```

Comment:
We should only cancel commands that require the subsystem being put in failsafe.

Reply:
Also, we should only cancel commands when the subsystem is being put into open loop. It is fine to keep running commands when going from open loop to closed loop.